src/log.ts

```diff
@@ -928,6 +923,16 @@ class Log implements LogSeverityFunctions {
         options.gaxOptions
       );
     }
+    function snakecaseKeys(labels: {[p: string]: string} | null | undefined) {
+      for (const key in labels) {
```

Comment:
please document what is happening in this function

Reply:
added!